stubs/networkx/networkx/testing/utils.pyi

```diff
@@ -0,0 +1,9 @@
+# Stubs for networkx.testing.utils (Python 3)
```

Comment:
I don't see the `testing` module in the source, did you mean `tests`? Either way, tests and other non-public APIs are preferably excluded from stubs, unless they are actually referenced by other public APIs.